test/test_client.py

```diff
@@ -926,10 +926,11 @@ def test_directory(self):
   def test_file(self):
     self.client.write('foo', 'hello, world!')
     self.client.set_acl('foo', 'user::rwx,user:foouser:rwx,group::r--,other::---')
+    self.client.modify_acl('foo', 'user:foouser:rw-')
     content = self.client.acl_status('foo')
     ok_(len(content) > 1)
     ok_(content['entries'] is not None)
-    ok_(any('user:foouser:rwx' in s for s in content['entries']))
+    ok_(any('user:foouser:rw-' in s for s in content['entries']))
```

Comment:
We're not testing the effect of the `set_acl` call anymore here. Let's keep the original test and add new ones for the added behavior.